docs/go_getting_started.md

```diff
@@ -101,10 +101,42 @@ anycable-go --port=443 -ssl_cert=path/to/ssl.cert -ssl_key=path/to/ssl.key
 => INFO time context=http Starting HTTPS server at 0.0.0.0:443
 ```
 
-## Docker
+## Deploying
+### Docker
 
 Official docker images available at [DockerHub](https://hub.docker.com/r/anycable/anycable-go/).
 
 ### Heroku
 
 See [heroku-anycable-go](https://github.com/anycable/heroku-anycable-go) buildpack.
+
+### Systemd
+
+To run on systemd you can:
+
+1. Edit as needed and save the following script to `/etc/systemd/system/anycable-go.service`
+2. Reload services via `sudo systemctl daemon-reload`
+3. Run as usual - `sudo systemctl start anycable-go`
+
+```
```

Comment:
Let's add syntax highlighting!
```suggestion
```ini
```
Not sure whether it will affect resulting documentation, but at least it will help at GitHub.

As per [docs](https://www.freedesktop.org/software/systemd/man/systemd.unit.html):
> A unit file is a plain text ini-style file

Reply:
It will not, but agree